.vscode/settings.json

```
@@ -1,10 +1,19 @@
 {
-  "eslint.workingDirectories": ["./"],
+  "eslint.workingDirectories": [
+    "./"
+  ],
   "solidity.compileUsingRemoteVersion": "v0.8.4+commit.c7e474f2",
   "solidity.solhintRules": {},
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "prettier.requireConfig": true,
   "solidity.formatter": "none",
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+  "files.watcherExclude": {
+    "**/.git/objects/**": true,
+    "**/.git/subtree-cache/**": true,
+    "**/node_modules/**": true,
+    "cache/**": true,
+  },
+  "git.autoRepositoryDetection": false
```

```suggestion
  "git.autoRepositoryDetection": "openEditors"
```
I think we want this